lib/ioki/model/operator/multilanguage_string.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      class MultilanguageString < Base
```

Comment:
MultiLanguageString

Reply:
But it's `MultilanguageStringSerializer` in triebwerk 🤔

Reply:
What does this class do? is it only a string? String as classname is .. well .. too generic

Reply:
This is the representation of the `MultilanguageString` model of triebwerk which can be found in the model concerns. For sake of keeping things consistent.

Reply:
As discussed let's keep this consistent with triebwerk 👍